contrib/jinzhu/gorm/gorm.go

```diff
@@ -7,17 +7,109 @@
 package gorm
 
 import (
+	"context"
+	"math"
+	"time"
+
 	sqltraced "gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 
 	"github.com/jinzhu/gorm"
 )
 
+const (
+	gormContextKey       = "dd-trace-go:context"
```

Comment:
Should we prefix these with `key*`. We generally tried to do that. Seems to be the common pattern in Go (e.g. net/http constants)

Reply:
I don't think it reads well to prefix this with `key`.

Reply:
I guess we already have a common prefix and that's good enough for me. 👍